### PR TITLE
Move files and add Firebase dependency

### PR DESCRIPTION
**Pull request by aroquev00**
Just moved our files to /starters and copied the starter files for our capstone. I also added the Firebase dependency to the POM.xml file.
AHHH also, do you guys mind using the port 8181 for the dev server? It's because the codeserver on my Cloudtop has taken over my 8080 and I don't know how to change that : /

### CODE REVIEW

File: cloudbuild.yaml [hunk 1]
```diff
@@ -1,11 +1,4 @@
...
     entrypoint: mvn
-    args: ['-f', './pamelalozano', 'package']
-  - name: maven
-    entrypoint: mvn
-    args: ['-f', './aroquev/portfolio', 'package']
-  - name: maven
-    entrypoint: mvn
-    args: ['-f', './ayup/portfolio', 'package']
-
+    args: ['-f', './', 'package']
```
Comment (seanjohnite) [inline cloudbuild.yaml:4]: Can you just add your starters/ directory to the top level here? looks like they are not running in the check

File: starters/cloudbuild.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  - name: maven
+    entrypoint: mvn
+    args: ['-f', './pamelalozano', 'package']
+  - name: maven
+    entrypoint: mvn
+    args: ['-f', './aroquev/portfolio', 'package']
+  - name: maven
+    entrypoint: mvn
+    args: ['-f', './ayup/portfolio', 'package']
+
```
Comment (seanjohnite) [inline starters/cloudbuild.yaml:11]: I don't think this is working with cloudbuild unfortunately :/

File: cloudbuild.yaml (hunk 1 above)
Comment (aroquev00) [reply cloudbuild.yaml:4]: Done.

File: starters/pamelalozano/src/main/webapp/WEB-INF/appengine-web.xml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
+  <threadsafe>false</threadsafe>
```
Comment (brettapeters) [inline starters/pamelalozano/src/main/webapp/WEB-INF/appengine-web.xml:3]: You can probably make this `<threadsafe>true</threadsafe>`

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+        <version>2.2.0</version>
+        <configuration>
+          <deploy.projectId>quizzy-step-2020</deploy.projectId>
+          <deploy.version>1</deploy.version>
+          <port>8181</port>
+        </configuration>
+      </plugin>
+    </plugins>
+  </build>
+</project>
```
Comment (brettapeters) [inline pom.xml:50]: nit. add newline

File: starters/cloudbuild.yaml (hunk 2 above)
Comment (brettapeters) [reply starters/cloudbuild.yaml]: Should we just stop running cloudbuild for starter projects?

File: starters/pamelalozano/src/main/webapp/WEB-INF/appengine-web.xml (hunk 3 above)
Comment (aroquev00) [reply starters/pamelalozano/src/main/webapp/WEB-INF/appengine-web.xml:3]: Done.

File: pom.xml (hunk 4 above)
Comment (aroquev00) [reply pom.xml:50]: Done.

File: starters/pamelalozano/src/main/webapp/WEB-INF/appengine-web.xml (hunk 3 above)
Comment (aroquev00) [reply starters/pamelalozano/src/main/webapp/WEB-INF/appengine-web.xml]: Done.

**Pull request merged**